File: other/Next Prime.py
import sys
from math import sqrt,floor
N=int(sys.stdin.readline())
def isprime(N):
    if N==0 or N==1: return False
    for x in range(2,floor(sqrt(N))+1):
        if N%x==0 and N!=x:
            return False
    return True
for x in range(N,20000000001):
    if x<0:
        continue
    elif isprime(x):
        print(x)
        sys.exit()

File: other/Next Prime.cpp
#include <iostream>
#include <math.h> 
using namespace std;
int isprime(int x){
  int g = static_cast <int> (floor(sqrt(x)));
  if(x<2){
    return false;
  }
  else{
    for(int i=2;i<=g;i++){
      if(x%i==0){
        return false;
      }
    }
  return true;
  }
}
int main()
{
  long y=20000000000;
  int N;
  scanf("%d",&N);
  for(int j=N;j<y;j++){
    if(isprime(j)==1){
      printf("%d",j);
      break;
    }
  }
}